boutdata/collect.py

```diff
@@ -270,14 +277,14 @@ def getDataFile(i):
     # Read data from the first file
     f = getDataFile(0)
 
+    dimensions = f.dimensions(varname)
```

Comment:
Any reasons to move this?

Reply:
Not that I can see - the original place looks more correct. Will move back.

Reply:
Also noticed that the `findVar()` bit is not included in the `parallel=True` branch further up, but I think it should be. Is only a minor fix, so I'll tack that onto this PR.